src/ethereum/osaka/vm/instructions/system.py

```diff
@@ -1271,3 +1379,58 @@ def return_contract(evm: Evm) -> None:
 
     # PROGRAM COUNTER
     evm.running = False
+
+
+def pay(evm: Evm) -> None:
```

Comment:
This is not part of current spec (April 15) https://github.com/ethereum/EIPs/blob/5fae72b226be42a043e75c682d13d3eb889da17b/EIPS/eip-5920.md but this should be rejected in the STATIC frame (when previously STATICCALL has been used in the call stack). Part of https://github.com/ethereum/EIPs/pull/9590